package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "next": "12.2.5",
     "next-seo": "^5.5.0",
+    "pnpm": "^7.9.5",
```

Comment:
Still to remove and as we won't install any new lib the `pnpm-lock.yaml` file should remain unchanged.

Reply:
Dona. Thanks for the comment.